dali/python/nvidia/dali/external_source.py

```diff
@@ -397,6 +474,36 @@ def __call__(self, *, source = None, cycle = None, name = None, layout = None, c
         if batch is None:
             batch = True
```

Comment:
How about:
```suggestion
        if batch is None:
            batch = not parallel
```
?

Reply:
I would prefer to keep the `batch` mode to be selected explicitly by the user.

Reply:
But that's forcing the user to type `parallel=True, batch=False` each time. Looking back, I'd prefer to have batch=False by default - but we can't do that now... or can we, in 1.0?

Reply:
That would break too many of already working pipelines with ExternalSource.
I can make it automatic with parallel.